### DISCUSSION TITLE
Questions about multicontrolnet

### DISCUSSION THREAD

**Discussion by alexblattner** [Mikubill/sd-webui-controlnet#454] [Q&A]
Hi, I was always curious about multicontrolnet and how it works. What happens behind the scenes? do you just run the first 2 images through the first model then take it to the next, etc... or is the process different?

Would greatly appreciate the answer thanks

**Comment by Petas3** [comment]
Also curious if the order of the nets is impacting results in any way.

**Comment by ljleb** [comment]
I'm a bit curious as well! I'll convert this to a discussion as this is not a bug report nor a feature request.
